README.md

```diff
@@ -98,63 +120,85 @@ To provide further configuration, add the `custom.kubeapilinter` section to your
 
 [golangci-config-struct]: https://pkg.go.dev/sigs.k8s.io/kube-api-linter/pkg/config#GolangCIConfig
 
-Where fixes are available within a rule, these can be applied automatically with the `--fix` flag.
+Where fixes are available within a rule, these can be applied automatically with the `--fix` flag:
 
 ```shell
-golangci-kube-api-linter run path/to/api/types --fix
+./bin/golangci-lint-kube-api-linter run path/to/api/types --fix
```

Comment:
I guess generally we expect people to sort their own path, maybe we don't need to prefix the bin here?